.github/workflows/scanner-offline-bundle-update.yaml

```diff
@@ -0,0 +1,64 @@
+name: Scanner release offline vulnerability bundle update
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+  workflow_dispatch:
+  push:
+    branches:
+    - scanner/*
+    - yli3/*
+
+jobs:
+  read-release-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.output-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Generate matrix JSON
+      id: output-matrix
+      run: |
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+        echo "matrix<<$EOF" >> "$GITHUB_OUTPUT"
+        # Extract X.Y versions, ensuring uniqueness
+        grep -oE '^[0-9]+\.[0-9]+' scanner/updater/version/RELEASE_VERSION | sort -u | jq -Rs '{ versions: ( sub("\n$"; "")|split("\n") ) }' | tee -a "$GITHUB_OUTPUT"
```

Comment:
Just to confirm: In #9754 , we added functionality that adds each new release and RC to the `RELEASE_VERSION` file. In this workflow, you only need X.Y, and in `scanner-release-vuln-update.yml` you iterate over all releases (incl. patches, release candidates)?

Reply:
Yeah, there was a design change for **_offline bundle_** from supporting X.Y.Z version to X.Y meanwhile we will ensure backward compatibility for all major versions (X.Y). This was a very late change in design before the release, sorry if there's any confusions. 

There's also a discussion in the consolidation channel on whether we should do the same for online vuln bundle but we haven't decided at the moment.